src/api/courses/fetchers/fetchPreviewCourse.ts

```diff
@@ -0,0 +1,8 @@
+import { Course } from '../../../types/courses/courses'
+import { wiseyStudy } from '../../index'
+
+interface Params {
+  courseId: string
+}
+
+export const fetchPreviewCourse = ({ courseId }: Params) => wiseyStudy.get<Course>(`/core/preview-courses/${courseId}`)
```

Comment:
It's better to avoid long lines in code:
```suggestion
export const fetchPreviewCourses = () => {
  return wiseyStudy.get<AllCourses>('/core/preview-courses');
};
```